config.yml

```diff
@@ -287,24 +287,8 @@ genesis:
               amount: "200000"
     incentive:
       params:
-        lp_incentives:
-          - eden_amount_per_year: "10000000000000"
-            distribution_start_block: "1"
-            total_blocks_per_year: "10512000"
-            allocation_epoch_in_blocks: "28800"
-            max_eden_per_allocation: "27397238400"
-            distribution_epoch_in_blocks: "10"
-            current_epoch_in_blocks: "0"
-            eden_boost_apr: "1"
-        stake_incentives:
-          - eden_amount_per_year: "10000000000000"
-            distribution_start_block: "1"
-            total_blocks_per_year: "10512000"
-            allocation_epoch_in_blocks: "28800"
-            max_eden_per_allocation: "27397238400"
-            distribution_epoch_in_blocks: "20"
-            current_epoch_in_blocks: "0"
-            eden_boost_apr: "1"
+        lp_incentives: []
+        stake_incentives: []
```

Comment:
@kenta-elys any reasons we are not getting rid of those params as you are using the ones from tokenomics now?

Reply:
@cosmic-vagabond I am still using the params in incentive module and filling in the value from tokenmocis module. Tokenmocis module params aren't enough to handle current incentive codebase. Let me think more if we can get rid of this.